cibuild.cmd

```diff
@@ -0,0 +1,6 @@
+"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" x86
```

Comment:
You need to prefix this with `call`.  Otherwise it just executes that batch file and exits.  

http://stackoverflow.com/questions/4798879/how-to-run-a-batch-script-from-within-a-batch-script